exercises/concept/the-farm/the_farm_test.go

```diff
@@ -14,192 +14,141 @@ func (wf testWeightFodder) FodderAmount() (float64, error) {
 	return wf.fodder, wf.err
 }
 
+var NonScaleError = errors.New("non-scale error")
+
 func TestDivideFood(t *testing.T) {
-	nonScaleError := errors.New("non-scale error")
 	tests := []struct {
-		description             string
-		weightFodder            WeightFodder
-		weightFodderDescription string
-		cows                    int
-		wantAmount              float64
-		wantErr                 error
+		description      string
+		fodderAmount     WeightFodder
+		cows             int
+		wantFodderAmount float64
+		wantFodderErr    error
 	}{
 		{
-			description:             "100 fodder for 10 cows",
-			weightFodder:            testWeightFodder{fodder: 100, err: nil},
-			weightFodderDescription: "100 fodder, no error",
-			cows:                    10,
-			wantAmount:              10,
-			wantErr:                 nil,
+			description:      "100 fodder for 10 cows with no error",
+			fodderAmount:     testWeightFodder{fodder: 100, err: nil},
```

Comment:
This field contains a fodder, not an amount.

Reply:
Do you think its a good idea to update `wantAmount` and `wantErr` to `fodderAmount` and `fodderError` ?